### DISCUSSION TITLE
기획 변경에 대한 의견

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#16] [아이디어]
기획 변경에 대한 의견을 근거와 함께 작성해주세요.

1. 현행 유지
2. 스터디 툴 서비스로 전환

**Comment by nlom0218** [comment]
### 현행 유지 + 진행 중에 우리가 제공할 수 있는 것 조금만한 무언가 추가

빠른 모집, 빠른 진행, 빠른 헤어짐 이것이 하루 스터디에서 제공하는 기능이다. 현재 MVP을 본다면 이를 충분히 구현할 수 있다고 생각함. 애초에 하루라는 짧은 시간만 제공하는 것이니까.

이런 빠른 것이 현재 우리 서비스의 뾰족함이라고 생각함. 이에 더해서 다른 서비스와 또다른 차이점을 제공하자면 다음과 같이 생각할 수 있음

1. 스터디장에게 스터디 진행을 도와줄 수 있는 무언가
2. 스터디원이 스터디가 끝난 후 얻어갈 수 있는 무언가

위의 두 가지에 대해 추가할 수 있는 조금만한 것을 함께 고민하고 구현만 해도 2차 스프린터에 보여줄 수 있는 것이 생길 수 있다고 생각함. 또한 3, 4차 스프린터를 위해 우리가 구체화할 내용이 명확히 그려진다고 봄.

난 욕심이 많아서 하고 싶은게 정말 많음ㅠㅠㅠ 

파이팅해서 잘 해보자구요 ღ •͈ᴗ•͈ ღ

**Comment by woosung1223** [comment]
1. 현행 유지

모집 플랫폼 제공이 '비타민'이기는 하나, `하루동안 스터디를 짧고 빠르게 한다`라는 개념을 위한 `진통제`이기도 하다. 만약 모집 기능이 없다면, 하루 동안 스터디를 하길 원하는 사람들은 어디서 스터디를 구할 것인가?
결국 기타 플랫폼(인프런, OKKY) 혹은 지인들에게 스터디를 하지 않겠냐고 물어볼텐데 과연 하루만 스터디를 할까? 
하루동안 스터디를 한다는 것은 스터디 패러다임의 새로운 전환인데, 이 패러다임에 사용자들이 익숙해지도록 제공하는 것이 모집 기능이라는 생각이 든다.

또한 기획을 트는 경우 데드라인을 지키지 못할 것이라는 생각도 든다. 
실무에서 비슷한 일이 생기는 경우 그때마다 기획을 갈아엎을 순 없다. 현재 주어진 것에서 최대한 제공할 수 있는 것을 제공해야 한다.

그리고 지금도 충분히 하루스터디만의 강점을 세울 수 있다고 본다! 킬링 피쳐들이 여럿 존재하기 때문(하루동안 스터디를 진행한다는 것, 스터디 주제를 키워드별로 가져간다는 것, 미정이지만 스터디 목표를 작성할 수 있는 템플릿을 제공하는 것)

**Reply by jaehee329** [reply]
우리는 모집 기능을 제공하긴 하지만 초기의 경우 플랫폼이 있어도 사용자 풀이 없으므로 초기 유입된 사용자가 `스터디를 구할 곳은 있는데 사람이 없더라` 라는 상황이 나올까 걱정돼.

> 결국 기타 플랫폼(인프런, OKKY) 혹은 지인들에게 스터디를 하지 않겠냐고 물어볼텐데 과연 하루만 스터디를 할까?

만약 툴이 제공된다면(기능은 협의해야겠지만) 5일치 스터디를 잘게 나눠 진행해보면서 우리가 제시하는 패러다임을 느껴볼 수 있지 않을까?

일정에 대해서는 논의해봐야 할 것 같아.

**Comment by jaehee329** [comment]
2. 스터디 툴로 새로운 기획

### 초기의 모집 기능만을 접한 사용자는 잔존할지 의문이다.
하루스터디의 가장 큰 가치는 `간단한 주제로 짧게 스터디를 진행, 성취를 빠르게 얻는 것`이라 생각.
이것은 새로운 패러다임인데 어떻게 사용자로 하여금 이것을 익숙해지고 지속적으로 사용케 할까?
준이 말했듯 새로운 방식을 도입했을 때 그 경험이 1. 어렵지 않아야 하고, 2. 유익해야 사용해 본 사용자가 잔존한다.

우리는 애자일 방법론을 적용하기로 했다.
조그마한 킥보드를 만들어 대중에게 빠르게 공개해야 하는데 지금까지 기획한 `모집 플랫폼`은 킥보드가 맞는지는 의문이 든다.
1. 스터디 플랫폼에는 항시 유지되는 사용자 풀이 필요하나 처음에는 이것이 존재하지 않는다.
2. 지인을 통해 사용자를 모아 서비스를 경험시켜도 단순히 모집 단계에서 목표와 할 일을 적는 정도로는 준이 제공한 구글 폼 정도의 유익한 경험을 느끼기는 힘들다.

당장 기존의 기획으로 프로토타입이 나오더라도 사용 범주는 우테코 내부일 것이라 예상되는데 이것은 사정을 다 아는 내부인의 알파 테스트라고 느껴진다.
기존에 의논했듯 스터디 진행 단계에 대한 개입이 없다면 `작은 단위 스터디의 방식과 가치`를 온전히 전달하긴 어렵다.
어떻게 진행해야 하는가에 대한 가이드가 없고 왜 써야하는지 알지 못한다는게 그 이유이며, 오래 의논했던 채팅방이 그 해소 창구.

이런 진행 단계에 대한 기능이 추가되어야 비로소 하루스터디가 전하려는 가치를 사용자들도 느끼게 될 것이며 이 시점이 내가 생각하는 킥보드이다.
만약 이런 기능 없이 지인들에게 권유하는 방식으로 사용자를 끌어들인다면 우리가 전달하는 가치와 유익함을 느끼기 어려울 것이고,
이후 기능이 추가되더라도 전에 써봤던 경험이 유익하지 않아 선뜻 다가오기 어렵다.
마치 예능에 출연하는 배우들이 연기자로써의 이미지를 소비해 그들이 출연하는 영화를 보기도 전에 예능의 이미지가 떠올라 몰입이 되지 않는 것처럼.

### 모집 기능을 미뤄놓고 스터디 툴부터 만들면 좋은 점

나는 준이 제공했던 폼을 기반으로, 여러 사용자가 동시에 접속해 폼을 채워나가고 타이머를 제공해 시간에 따라 스터디가 성공적이었는지 폼을 다시 채우도록 하는 툴을 생각하고 있다.

방장이 세션을 만들고 지인들 중심으로 간단히 폼과 함께 스터디를 진행하는 방식이므로 우리가 전달하려는 작은 단위 스터디라는 패러다임의 장점을 먼저 느끼도록 할 수 있다.

또한 사용자 풀이 없어도 된다는 것도 정말 큰 장점이다.
스터디 모집 플랫폼을 만들어도 참여자가 없다면 죽은 플랫폼이고 초기 사용자를 모집하는 것이 너무 어렵지만,
지인을 기반으로 한 사용자들을 위한 툴로 시작하면 하루스터디에 익숙한 사용자가 플랫폼으로 넘어오는 심리적 장벽이 낮아진 상태일 것이다.

즉,
`지인들과 사용해보니 좋았는데` -> `이런 툴 기반이면 임의의 타인과 해도 좋을 것 같아서 모집 플랫폼에도 유입`이
`전에 써봤을땐 어떻게 스터디해야하는지 몰랐는데` -> `다시 들어가보니 툴도 제공되서 한번 더 도전` 보다 쉽다고 생각한다.


### 일정과 관련하여
아까도 말했듯 다른 팀들도 크게 기획이 변경될 수 있는 상황에서 모집 기능 기획을 아카이빙만 해두고 새로운 기획으로 툴부터 제작하는 것이 크게 늦어진다고 생각이 들진 않는다.

만약 모집으로 출시한다면 피드백으로 받게 될 것은 `모집을 하긴 했는데 어떻게 스터디하는건지 잘 모르겠다`라는 말로 지금도 예상되며,
1. 2차 스프린트까지 모집 플랫폼을 만들어 피드백 기반으로 3차 스프린트에 스터디 진행 기능을 넣는 것보다
2. 이미 기획을 한번 경험한 우리가 2차 스프린트 내에 빠르게 간단한 툴을 기획하고 기본 기능을 구현하고 3차 이후 모집을 붙이는 편이

큰 그림에서는 결과물은 비슷하나 대중에게 공개 가능한 결과물이 더 빠르게 나오고 대중의 피드백도 더 유용할 것이라 예상된다.


---
모집으로 가닥이 정해져도 괜찮으나 2차 스프린트까지의 결과물이 완전 일반인에게 공개하고 피드백을 받을 수 있는 수준일지는 의문이며
당장은 늦어지지만 결과적인 완성도는 비슷하고, 툴이 먼저 제공되는 것이 피드백이나 사용자 경험이 좋을 것이라는 입장이다.

**Comment by woo-jk** [comment]
2. 스터디 툴 서비스로 전환

근로 회의 진행하면서 복잡했던 제 생각을 조금 정리했는데 참고만 부탁드릴게요.

### 2번이 중요한 이유
과연 모집 플랫폼을 제공하는 것이 다른 서비스들과의 차별성이 뚜렷할지 의문이 든다고 생각합니다. 
단순히 기존 스터디 플랫폼에서 모집 공고에 `우리 스터디는 XXX 키워드를 단 하루만 스터디 할거니 빠른 스터디를 원하는 사람들은 참여하세요` 이렇게 한 줄 적으면 끝 아닌가?라는 생각이 듭니다.

기존 플랫폼이 인원도 훨씬 많고, 활성화도 훨씬 잘 되어있으니 하루 동안 스터디를 하고 싶은 사람은 우리 서비스를 이용하기 보단 위와 같은 방식으로 기존 스터디 서비스를 이용할 것 같습니다.
그렇게 된다면 우리 서비스는 '진짜 사용자를 위한 서비스'가 아닌 단순 포트폴리오용 서비스로 전락할 가능성이 크겠죠..ㅠ

우리의 서비스는 사용자에게 새로운 패러다임을 제시하는 것이라고 생각합니다.
진행 관련해서 기존에 없던 진행 방식을 단순 느낌만 제공하는 것이 아닌, '제대로' 제공을 한다면 우리 서비스의 차별성은 훨씬!! 커진다고 생각합니다.

또한 하루스터디 네이밍에 대해서 말하자면, 
스프레드 시트의 내용을 보면 '하루 동안' 어떤 주제를 가지고 공부 할 것인지 먼저 정하고 시작하는데, 이런 플로우가 있다면 하루스터디 라는 네이밍은 퇴색되지 않는 것 같습니다.

### 2번을 먼저 해야하는 이유

모집 플랫폼은 CRUD 기능 위주이고,진행 툴은 소켓?을 통한 실시간 상호작용 위주로 갈 것 같습니다. 두 가지 기능을 모두 구현하면 정말 정말 좋겠지만.. 과연 현실적으로 본다면 레벨3 안에 이 두가지 기능 모두 `퀄리티 있게` 구현이 가능할까?라는 생각이 문득 듭니다.

기존 모집 플랫폼을 계속 가면 로그인 기능, 공고 및 참여에 대한 기능 등 여러 기능을 구현해야 하는데 , 특히 `로그인 기능`과 관련된 여러 파생 기능들이 시간이 꽤 걸릴것 같습니다..

데모데이 때 구구가 2차 데모데이까지의 기능 분량에 대해 쎄게 말한 것도 있지만 어느정도 맞는 말 한거라고 생각합니다. 다른 팀들 보면 2차 데모데이까지 기능 한 두 개 확실하게 하는 정도에 그치는 수준이었습니다.

우리가 레벨1, 2동안 미션 할 때 했던 것과 우리의 실제 진행하는 프로젝트는 분명 다르다고 생각합니다. 여러 가지 고려해야 할 것도 많을 것이고 우리 생각보다 빠르게 기능 구현이 되지 않을 것이라 생각합니다. 정말 중요한 기능인 진행 도구 기능을 먼저 확실하게 정하고 모집 기능은 나중에 진행하면 어떨까요?

**Reply by jaehee329** [reply]
저도 비슷하게 생각하는게,
당장은 차별화가 되는 것은 `스터디를 시스템적으로 하루만 가능하게 한다`인데 모집 자체의 과정이 쉽진 않은 상황에서 
하루만 스터디하는 것의 필요성을 모르는 사람들이 `왜 하루만 스터디를 해야 해?` 라는 의문이 더 많이 떠오를 것 같아요.
처음 그 패러다임이 얼마나 유익한지 느껴보기도 전에요.

우리의 서비스가 포트폴리오용 서비스일지, 진짜 이 가치를 전달하고자 하는 건지도 고민해봐야겠네요.

**Comment by aak2075** [comment]
1. 현행 유지

진행을 도와주는 툴이라는 것이 당장은 너무 추상적이다.
또한 과연 이런 툴이 정말 사용자가 필요한 게 맞는가?
(굳이 툴을 만들기 보다 스프레드시트 링크를 제공하거나 가이드라인 정도만 제공해도 충분할수 있다)
우리는 하루만 스터디하는 패러다임을 제시하는 것이기 때문에 가이드 라인만 제시하고 툴이나 진행쪽으로 가려면 화상 회의같은 것이 훨씬 더 사용자가 필요한 기능이지 않을까 하는 생각이 든다.

진행 툴을 제공한다고 해도 멋들어진 무언가가 필요할지도 의문. 
그냥 스터디 상세 페이지에서 표 형태로 준의 스프레드 시트처럼 입력을 받아도 되는거 아닌가?
어떠한 툴을 만들고 기능을 추가하느라(타이머 라던가...) 쏟은 노력에 비해 그만큼 큰 의미가 있을지 의문이 든다.

모집 플랫폼으로써 우리 패러다임을 잘 사용하기 위한 가이드라인만 잘 제시하는 쪽으로 MVP를 정하고, 이후에 추가로 진행 툴을 추가해도 늦지 않는다고 생각한다.

**Comment by yeopto** [comment]
### 현행 유지

우리 서비스의 핵심 가치는, 하루만에 스터디를 마무리 지어서 반복적인 짧은 스터디를 통해 성취에 대한 효능감을 주는 것이다.

그렇다면 기존 기획 방향에서 효능감을 주기위해 효율적인 방법을 추가적으로 제시할 수 있다고 생각한다.

이 방법이 우리가 지금까지 해왔던 것들에 대해 이어갈 수 있다고 생각한다.

새로운 방향(효능감을 주기 위한 툴)으로 간다면 리소스가 많이 들 것 같다. 기획부터 해서 의사조율 등

또, 툴로 간다면 하루만에 끝낼 수 있다는 하루스터디 이름에 어울리지 않는 프로젝트이지 않나라는 생각을 한다.

툴을 만들고 모집을 붙이는 것은 좀 주객전도가 아닌가 싶다.

모집 부터해서 모집한 사람들이 효능감을 느낄 수 있는 기능을 추가하는 것이 맞는 방향인 것 같다.
